### DISCUSSION TITLE
Add support for Bluesky

### DISCUSSION THREAD

**Discussion by philippemnoel** [jordwest/news-feed-eradicator#370] [Ideas]
It's the new Twitter/X, apparently. I'd love its feed disabled please!

**Comment by yolfer** [comment]
+1, I would also appreciate this!

**Comment by ThairlanAnjos** [comment]
+1

**Comment by rouelibre1** [comment]
Bluesky support would be awesome

**Comment by jordwest** [comment]
added in ae45213d49f4e2c57bbac393320fe57c4617613d
